### PR TITLE
Adds hardcoded timeseries

### PR DESCRIPTION
**Pull request by briafassler9**
image

I'm debating on not including 2020 since the year isn't over yet, and screws the data in a big way since the total is not complete.

### CODE REVIEW

File: portfolio/src/main/webapp/js/custom-script.js [hunk 1]
```diff
@@ -600,6 +538,38 @@ function drawCharts() {
...
+  /** Fetches police killings data and uses it to create a chart. */
+  function drawTimeSeriesChart() {
+    fetch('/chart-data')
+        .then((response) => response.json())
+        .then((policeKillings) => {
+          const data = new google.visualization.DataTable();
+          data.addColumn('string', 'Year');
+          data.addColumn('number', 'People');
+          policeKillings.forEach((dataPair) => {
+            data.addRow([dataPair.year.toString(), dataPair.amountKilled]);
```
Comment (Jaxobc) [inline portfolio/src/main/webapp/js/custom-script.js]: You can make year a string in dataPair.

File: portfolio/src/main/webapp/js/custom-script.js [hunk 2]
```diff
@@ -570,9 +502,15 @@ async function loadChartData() {
   return chartData;
 }
 
+/**
+ * calls the timeseries and interactive chart, and fetches the data from
+ * their respective servlets.
+ */
+
```
Comment (Jaxobc) [inline portfolio/src/main/webapp/js/custom-script.js]: You can remove line 509.

**Review by Jaxobc — APPROVED** [review]

File: portfolio/src/main/webapp/js/custom-script.js (hunk 2 above)
Comment (briafassler9) [reply portfolio/src/main/webapp/js/custom-script.js]: done

File: portfolio/src/main/webapp/js/custom-script.js (hunk 1 above)
Comment (briafassler9) [reply portfolio/src/main/webapp/js/custom-script.js]: done

File: portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+import javax.servlet.http.HttpServletResponse;
+
+/* Servlet to read chart data from data store and return it in doGet */
+// TODO(briafassler): Fix the problem where chart only appears after pressing submit
+@WebServlet("/chart-data")
+public class ChartServlet extends HttpServlet {
+  private static final String TABLE_NAME = "Police Killings By Year";
+  private static final String YEAR = "year";
+  private static final String AMOUNT_KILLED = "amount killed";
+  private static final String test = "{'2020': '5'}";
```
Comment (kofimeighan) [inline portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java:50]: Is there a reason this is in production code?

File: portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    Query query = new Query(TABLE_NAME);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    List<DataPair> dataPairs = new ArrayList<DataPair>();
+    for (Entity entity : results.asIterable()) {
+      int amountKilled = (int) entity.getProperty(AMOUNT_KILLED);
```
Comment (kofimeighan) [inline portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java:60]: this is probably going to throw you an error. retrieving properties from the datastore that arent strings makes complications when casting

**Review by kofimeighan — APPROVED** [review]

**Review by kennzoid — APPROVED** [review]

**Comment by kennzoid** [comment]
Maybe we should remove the 2020 data point since it is a little misleading.

File: portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java (hunk 3 above)
Comment (briafassler9) [reply portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java:50]: it's been removed now

File: portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java (hunk 4 above)
Comment (briafassler9) [reply portfolio/src/main/java/com/google/sps/servlets/ChartServlet.java:60]: me and kenneth were able to fix the problem

**Pull request merged**